### ISSUE TITLE
top element "[..]"

### ISSUE THREAD

**Issue by omm** [rjopek/harbour-commander#3]
Can you explain this code
```
   IF cName == ".."
      cFileName := PadR( "[" + AllTrim( cFileName ) + "]" + Space( nLengthName ), nLengthName ) + " "
   ENDIF
```
why not use here just
```
IF cName == ".."
      cFileName := "[..]"
ENDIF
```
because if first element in subdir is directory with length is 3, the new top element showns as "[.."

**Comment by rjopek** [comment]
Try to change:
old code 
iif( nLengthName == 2, nLengthName := 4, nLengthName ) // ?
no 
iif( nLengthName == 2, nLengthName := 5, nLengthName ) // ?

This is how the low priority error is marked // ?

Length alignment
cFileName := PadR( "[" + AllTrim( cFileName ) + "]" + Space( nLengthName ), nLengthName ) + " "

**Comment by omm** [comment]
Try to enter to harbour-commander\docs and you see "[.." as top element, because "docs" has one dir "img" and it's length not 2 like in code iif (nLengthName == 2, ...)

**Comment by rjopek** [comment]
Yes,  
something clearly to be corrected.

- Rafał

**Comment by omm** [comment]
Fixed after [f2b54cc](https://github.com/rjopek/harbour-commander/commit/f2b54cc81a5c23b7a925be60a9315e09fa7faf8f)
